### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by diyanei**
Hi @jeroenterheerdt  and HAsmartirrigation devs !

Here is my PR for french translation.
I struggled and failed to check the french result on my HA devcontainer: only the configuration workflow got translated for whatever reason, so I might need some tweak in the future to get it smooth (blind translation is not easy).

Anyhow, I also fixed a small typo in services.yaml.

Thank you for your development, I use it daily for 2 years now on my terrasse for pots irrigation. 
By the way, do you have some recommendations on the configuration tweaks I might use to this specific use case ?

### CODE REVIEW

**Comment by jeroenterheerdt** [comment]
@diyanei further updates: maximum bucket, localized help page and extra explanation line as well

File: custom_components/smart_irrigation/const.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 CUSTOM_COMPONENTS = "custom_components"
 
 LANGUAGE_FILES_DIR = "translations"
-SUPPORTED_LANGUAGES = ["en","nl"]
+SUPPORTED_LANGUAGES = ["en","nl","fr"]
```
Comment (jeroenterheerdt) [inline custom_components/smart_irrigation/const.py:12]: this should also include "de"

**Review by jeroenterheerdt — CHANGES_REQUESTED** [review]

**Comment by jeroenterheerdt** [comment]
> 

thanks for your efforts! Please catch up the language files to the latest beta and I can roll it into a release so you can actually see the results of your work ;).
Regarding your use case, I'd recommend going to the community forum or using the Discussions on github to dive deeper into this. Let's use PR's for what they are for ;)

**Comment by jeroenterheerdt** [comment]
I have cherry picked your changes to the French language files (2x) and services.yaml. Closing this PR now as you didn't rebase before making the PR. No worries, it happens. Your changes should be reflected in the next beta though! Thanks!

**Comment by diyanei** [comment]
Thanks @jeroenterheerdt ! Just did a check on the result, and it is pretty solid. I might have a few slight fixes to propose, will do it when more changes will be needed.

**Pull request closed without merging**